pyk/src/pyk/ktool/kompile.py

```diff
@@ -28,6 +28,7 @@ def kompile(
     hook_namespaces: Iterable[str] = (),
     emit_json=False,
     concrete_rules: Iterable[str] = (),
+    additional_args: Iterable[str] = ()
```

Comment:
This is a matter of personal preference, but I like to add the trailing comma to such lists. It is easier to extend, rearrange, etc. this way.

```suggestion
    additional_args: Iterable[str] = (),
```

Reply:
My thinking here was that this should always be the last argument irrespective.  But it makes sense for consistency.